### DISCUSSION TITLE
Forward only photos/videos specifically from a group

### DISCUSSION THREAD

**Discussion by ThatDevCoder** [MrMissx/Telegram_Forwarder#19] [General]
First of all, thank you for the script!
Can you please create a feature that can specifically send photos or videos from a particular group? Because there is a lot of unnecessary stuff also forwarded from the group which we are forwarding.
And can you also create a feature that will only forward messages without a caption. That would be great if you could do that!

**Comment by MrMissx** [comment]
Hi. If you want to make the script only trigger with a certain messages you can edit / add more filter to the `MessageHandler`  in `auto_forward.py`.
As you can see in PTB docs of [`filters`](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.filters.html#module-telegram.ext.filters), you can add those filters to allow only subset of messages to pass to the handler.

E.G. in your first case you can add `Filters.photo  & Filters.video` so it won't trigger on non photo/video messages.
